### DISCUSSION TITLE
Welcome to twind Discussions!

### DISCUSSION THREAD

**Discussion by sastan** [tw-in-js/twind#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by itsMapleLeaf** [comment]
Hey 😄 I'm trialing out this library as a long time (ish) user of Tailwind and twin.macro. I see the promise, and I'm gonna raise any issues that come up along the way that I can't resolve myself
